apps/cnp/sbox/base/descheduler2.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: descheduler2
+  namespace: cnp
+spec:
+  timeout: 20m
+  releaseName: descheduler
+  chart:
+    spec:
+      chart: descheduler
+      version: 0.18.0
+      sourceRef:
+        kind: HelmRepository
+        name: descheduler
+        namespace: cnp
+  interval: 1m
+  values:
+    kind: CronJob
+    image:
+      repository: registry.k8s.io/descheduler/descheduler
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 500m
+        memory: 256Mi
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+      privileged: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      runAsUser: 1000
+    cronJobApiVersion: "batch/v1"
+    schedule: "*/3 * * * *"
+    suspend: false
+    deschedulingInterval: 3m
+    replicas: 1
+    command:
+      - "/bin/descheduler"
+    cmdOptions:
+      v: 3
+    deschedulerPolicyAPIVersion: "descheduler/v1alpha2"
+    deschedulerPolicy:
+      strategies:
+        profiles:
+          - name: ProfileName
+            pluginConfig:
+              - name: "RemovePodsViolatingTopologySpreadConstraint"
+            plugins:
+              balance:
+                enabled:
+                  - "RemovePodsViolatingTopologySpreadConstraint"
+    priorityClassName: system-cluster-critical
+    rbac:
+      create: true
+    serviceAccount:
+      create: true
+    livenessProbe:
+      failureThreshold: 3
+      httpGet:
+        path: /healthz
+        port: 10258
+        scheme: HTTPS
+      initialDelaySeconds: 3
+      periodSeconds: 10
+    service:
+      enabled: false
+    serviceMonitor:
+      enabled: false
+      insecureSkipVerify: true
+      serverName: null
```

Comment:
Can you check which of these are defaults and remove duplicates? A lot of config to look through for stuff pre-defined upstream the same way